### DISCUSSION TITLE
about CSRs

### DISCUSSION THREAD

**Discussion by matchchen** [mortbopet/Ripes#116] [Q&A]
Hi,
These project is very cool！
There‘s one question，I found GPRs but no CSRs in all views. I wonder if I miss something since CSRs are so important.

**Comment by mortbopet** [comment]
Correct, CSRs and the CSR-related instructions have not been implemented in the processor models. I've added an issue to track this if someone would like to pick it up https://github.com/mortbopet/Ripes/issues/117
